### DISCUSSION TITLE
Change how key is displayed

### DISCUSSION THREAD

**Discussion by BenVerdiman** [TarekRaafat/autoComplete.js#329] [Q&A] [answered]
Hello, I want to change how key is displayed.
For an example:
I have this json file:
`[
  {
    "name": "Peter Jonathon"
  },
  {
    "name": "John"
  }
]`

And when I search Peter in the input, it shows the full name (Peter Jonathon). I want to show only Peter, and to autocomplete with the full name.

Thanks in advance!

**Comment by folknor** [comment, marked as answer]
Yes so just use https://tarekraafat.github.io/autoComplete.js/#/configuration?id=element-optional-1

```js
new autoComplete({
	resultItem: {
		element: (element, data) => {
			element.innerHTML = `<span>${data.value.firstname}</span>`
		}
	}
})
```

**Comment by BenVerdiman** [comment]
Thanks!
